v2_MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,196 @@
+# Migrating from Delegates to Completion Handlers
+
+## Overview
+Version 2.0-beta of the SDK transitions from the delgate-based flows to completion handler-based flows. This change simplifies the integration and provides better compatibility with modern async/await patterns.
+
+### Key Changes
+
+### Important Change: Cancellation Handling
+In v2.0-beta, cancellations (e.g., 3DS cancellations, PayPal web flow cancellations) are now returned as errors rather than as separate delegate methods. There are new helper methods, static functions, to help you discern threeDSecure cancellation errors and PayPal web flow cancellation errors. 
```

Comment:
```suggestion
In v2.0, cancellations (e.g., 3DS cancellations, PayPal web flow cancellations) are now returned as errors rather than as separate delegate methods. There are new helper methods and static functions to help you discern threeDSecure cancellation errors and PayPal web flow cancellation errors. 
```
I don't think we need to mention `beta` in the `MIGRATION_GUIDE` since this guide primarily for when we go GA.

Reply:
that makes sense. And on the "and", I wanted to clarify that the methods are static functions.
Maybe it could be phrased as "There are new helper static functions.."

Reply:
Yup that makes sense. It flows nicely without extra punctuation.